### PR TITLE
audit_results: constant names made less confusing and shorter

### PR DESCRIPTION
**Pull request by ndushay**
(again, probably everyone should see this)

AuditResults constant names were confusing me:
- "OBJECT" referring to db vs. moab.
- "actual" which I neglected to migrate to the constants
- THE_INCREDIBLY_LONG_NAMES_I_WROTE_WERE_BOTHERING_ME

Connects to #539

### CODE REVIEW

File: app/services/audit_results.rb [hunk 1]
```diff
@@ -14,12 +14,12 @@ class AuditResults
 
   INVALID_ARGUMENTS = :invalid_arguments
   VERSION_MATCHES = :version_matches
-  ARG_VERSION_GREATER_THAN_DB_OBJECT = :arg_version_greater_than_db
-  ARG_VERSION_LESS_THAN_DB_OBJECT = :arg_version_less_than_db
+  ACTUAL_VERS_GT_DB_OBJ = :arg_version_greater_than_db
+  ACTUAL_VERS_LT_DB_OBJ = :arg_version_less_than_db
   CREATED_NEW_OBJECT = :created_new_object
```
Comment (tallenaz) [inline app/services/audit_results.rb]: `ARG` changes to `ACTUAL` on the left, but `arg` stays `arg` on the right?

File: app/services/audit_results.rb [hunk 2]
```diff
@@ -14,12 +14,12 @@ class AuditResults
...
   DB_UPDATE_FAILED = :db_update_failed
-  OBJECT_ALREADY_EXISTS = :object_already_exists
-  OBJECT_DOES_NOT_EXIST = :object_does_not_exist
+  DB_OBJ_ALREADY_EXISTS = :object_already_exists
+  DB_OBJ_DOES_NOT_EXIST = :object_does_not_exist
   PC_STATUS_CHANGED = :pc_status_changed
```
Comment (tallenaz) [inline app/services/audit_results.rb]: Similarly, `OBJECT` changes to `DB_OBJ` on the left, but `object` stays `object` on the right?

File: app/services/audit_results.rb (hunk 1 above)
Comment (ndushay) [reply app/services/audit_results.rb]: hahahaha - oops.  thanks.

File: app/services/audit_results.rb (hunk 2 above)
Comment (ndushay) [reply app/services/audit_results.rb]: also ... oops.  Will fix.

**Comment by ndushay** [comment]
@tallenaz fixed!  thanks for spotting it.

**Pull request merged**